src/modules/rppt_tensor_color_augmentations.cpp

```diff
@@ -669,6 +669,72 @@ RppStatus rppt_lut_host(RppPtr_t srcPtr,
     return RPP_SUCCESS;
 }
 
+/******************** color_temperature ********************/
+
+RppStatus rppt_color_temperature_host(RppPtr_t srcPtr,
+                                      RpptDescPtr srcDescPtr,
+                                      RppPtr_t dstPtr,
+                                      RpptDescPtr dstDescPtr,
+                                      Rpp8s *adjustmentValueTensor,
+                                      RpptROIPtr roiTensorPtrSrc,
+                                      RpptRoiType roiType,
+                                      rppHandle_t rppHandle)
+{
+    if (srcDescPtr->c != 3)
+    {
+        return RPP_ERROR_INVALID_ARGUMENTS;
```

Comment:
Lets use an error code that specifically points to channels. Check rpp_defs.h for alternative option, or introduce one like RPP_ERROR_INVALID_CHANNELS.

Reply:
Done